Integrations/AWS-CloudTrail/AWS-CloudTrail.yml

```
@@ -0,0 +1,644 @@
+category: IT Services
+commonfields:
+  id: AWS - CloudTrail--DEV
+  version: -1
+configuration:
+- display: Default Region
+  name: defaultRegion
+  required: true
+  type: 0
+- display: Role Arn
+  name: roleArn
+  required: false
+  type: 0
+- display: Role Session Name
+  name: roleSessionName
+  required: false
+  type: 0
+- display: Role Session Duration
+  name: sessionDuration
+  required: false
+  type: 0
```

May as well support this while we are in here.

```suggestion
  type: 0
- display: Access Key
  name: access_key
  required: false
  type: 0
- display: Secret Key
  name: secret_key
  required: false
  type: 4
- display: Trust any cert (Not Secure)
  name: insecure
  required: false
  type: 8
- display: Use system proxy
  name: proxy
  required: false
  type: 8
```